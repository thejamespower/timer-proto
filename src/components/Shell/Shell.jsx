import React from 'react'
import { Provider } from 'react-redux'

import store from '../../store'
import App from '../App'

const Shell = () => (
  <Provider store={store}>
    <App />
  </Provider>
)

export default Shell
