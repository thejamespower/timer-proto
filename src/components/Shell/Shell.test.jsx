import React from 'react'
import renderer from 'react-test-renderer'
import Shell from './Shell'

jest.mock('../App', () => 'App')

describe('Shell', () => {
  it('renders', () => {
    const component = renderer.create(
      <Shell />,
    )
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
