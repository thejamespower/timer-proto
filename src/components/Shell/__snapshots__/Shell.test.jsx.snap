// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shell renders 1`] = `<App />`;
